From 3cd3e18ef5743d3f78fa540af1141db8de1fdbcc Mon Sep 17 00:00:00 2001
From: Patrick Williams <patrick@stwcx.xyz>
Date: Fri, 5 Aug 2016 16:02:37 -0500
Subject: [PATCH] external: Use more-standard PREFIX vs prefix

Distro packaging tools typically use PREFIX instead of prefix,
so change all references to lower-case prefix to upper-case.

pflash uses PREFIX since at least b2649b.

Signed-off-by: Patrick Williams <patrick@stwcx.xyz>
---
 external/gard/rules.mk        | 6 +++---
 external/opal-prd/Makefile    | 6 +++---
 external/xscom-utils/Makefile | 6 +++---
 3 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/external/gard/rules.mk b/external/gard/rules.mk
index 0dd14ed..dfe93d5 100644
--- a/external/gard/rules.mk
+++ b/external/gard/rules.mk
@@ -7,9 +7,9 @@ OBJS     += $(LIBFLASH_OBJS)
 OBJS     += common-arch_flash.o
 EXE       = gard
 
-prefix = /usr/local/
-sbindir = $(prefix)/sbin
-datadir = $(prefix)/share
+PREFIX = /usr/local/
+sbindir = $(PREFIX)/sbin
+datadir = $(PREFIX)/share
 mandir = $(datadir)/man
 
 GARD_VERSION ?= $(shell ./make_version.sh $(EXE))
diff --git a/external/opal-prd/Makefile b/external/opal-prd/Makefile
index 7482af3..8318348 100644
--- a/external/opal-prd/Makefile
+++ b/external/opal-prd/Makefile
@@ -3,9 +3,9 @@ LDFLAGS += -m64
 ASFLAGS = -m64
 CPPFLAGS += -I. -I../../include -I../../
 
-prefix = /usr/local/
-sbindir = $(prefix)/sbin
-datadir = $(prefix)/share
+PREFIX = /usr/local/
+sbindir = $(PREFIX)/sbin
+datadir = $(PREFIX)/share
 mandir = $(datadir)/man
 
 all: links arch_links | opal-prd
diff --git a/external/xscom-utils/Makefile b/external/xscom-utils/Makefile
index 3fee247..9695105 100644
--- a/external/xscom-utils/Makefile
+++ b/external/xscom-utils/Makefile
@@ -3,9 +3,9 @@ CC = $(CROSS_COMPILE)gcc
 XSCOM_VERSION ?= $(shell ../../make_version.sh xscom-utils)
 CFLAGS += -O2 -g -Wall -m64
 
-prefix = /usr/local/
-sbindir = $(prefix)/sbin
-datadir = $(prefix)/share
+PREFIX = /usr/local/
+sbindir = $(PREFIX)/sbin
+datadir = $(PREFIX)/share
 mandir = $(datadir)/man
 
 %.o: %.c
-- 
2.9.0

